ansible/roles/util-cfg-service/templates/terrain.properties.j2

```diff
@@ -58,8 +58,9 @@ terrain.jex.base-url = {{ jex.base }}/
 # Tree viewer settings
 terrain.tree-viewer.base-url = {{ tree_parser_base }}
 
-# Elastic Search settings
-terrain.infosquito.es-url = {{ elasticsearch.base }}
+# Elasticsearch settings
+terrain.infosquito.es-url   = {{ elasticsearch.base }}
+terrain.infosquito.es-index = {{ elasticsearch.data_index }}
```

Comment:
It's a bit weird that these are prefixed "infosquito" but I'm not really concerned about it so I kept it for consistency.